### ISSUE TITLE
v0.4.4 for fabric 1.19.2 has the forge version of create as a dependency

### ISSUE THREAD

**Issue by calliecameron** [alexandsr/BellsAndWhistlesMod#30]
The latest version of Create is 0.5.1.f. The 1.19.2 fabric version of this mod still depends on Create 0.5.1.c.

**Comment by calliecameron** [comment]
I tried installing it anyway and it seems to work with 0.5.1.f.

**Comment by a0ax** [comment]
Yea I was gonna say
Think I left the version range open

**Comment by calliecameron** [comment]
I looked into this again, and I think the problem is actually with the metadata on modrinth: when I install the latest fabric 1.19.2 version (0.4.4), it suggests the forge version of create as a dependency, i.e. [this](https://modrinth.com/mod/create) instead of [this](https://modrinth.com/mod/create-fabric). Looks like 0.4.2 has the right dependency.

I was initially confused by it asking to install create when it was already installed, and assumed it was due to asking for 0.5.1.c instead of 0.5.1.f, not realising it was actually a forge/fabric issue.
